src/Behat/MinkExtension/ServiceContainer/Driver/AppiumFactory.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of the Behat MinkExtension.
+ * (c) Shashikant jagtap <shashikant.jagtap@aol.co.uk>
```

Comment:
This is wrong; the copyright should be the one of the library, not your own name.